source/kotlin/src/main/kotlin/io/github/cosinekitty/astronomy/astronomy.kt

```diff
@@ -1028,32 +1028,26 @@ data class StateVector(
  * The positions are expressed in astronomical units (AU),
  * and the velocities in AU/day.
  */
-class JupiterMoonsInfo(
-    /**
-     * An array of state vectors for each of the 4 moons, in the following order:
-     * 0 = Io, 1 = Europa, 2 = Ganymede, 3 = Callisto.
-     */
-    val moon: Array<StateVector>
-) {
+class JupiterMoonsInfo(time: Time) {
```

Comment:
Just as a suggestion, this way creation of jupiterMoons will be merged with JupiterMoonsInfo, or maybe moons info can be provide as a method to Body.Jupitor enum methods